go/pkg/sysdb/coordinator/table_catalog.go

```diff
@@ -967,11 +967,7 @@ func (tc *Catalog) ForkCollection(ctx context.Context, forkCollection *model.For
 		if err != nil {
 			return err
 		}
-		// NOTE: This is a temporary hardcoded limit for the size of the lineage file
-		// TODO: Load the limit value from quota / scorecard, and/or improve the lineage file design to avoid large lineage file
```

Comment:
I think it is valid for this to still exist as backstop validation and defensiveness.

Reply:
I'll reimplement